compose/compiler/compiler-hosted/src/main/java/androidx/compose/compiler/plugins/kotlin/lower/ComposableFunctionBodyTransformer.kt

```diff
@@ -3647,12 +3656,14 @@ class ComposableFunctionBodyTransformer(
             private var shouldRealizeCoalescableChild = false
             private var coalescableChild: BlockScope? = null
         }
+
```

Comment:
same thing here and other places if you don't mind: let's try to keep the diff only to what code has actually changed